protocol/lavasession/single_consumer_session.go

```diff
@@ -151,3 +152,21 @@ func (consumerSession *SingleConsumerSession) VerifyLock() error {
 	}
 	return nil
 }
+
+func (scs *SingleConsumerSession) VerifyProviderUniqueId(providerUniqueId string) bool {
```

Comment:
its not just veify its store please rename

Reply:
Fix in 968bfe486caab617e4c923145e596bc9d269f721